<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>488</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="connectionLabel">
         <property name="text">
          <string>Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="connectionTypeLabel">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="connectionType">
           <item>
            <property name="text">
             <string>Any</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Localhost</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IP Address</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="connectionIPLabel">
           <property name="text">
            <string>IP Address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="connectionIP"/>
         </item>
         <item>
          <widget class="QLabel" name="connectionPortLabel">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="connectionPort">
           <property name="minimum">
            <number>1000</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="databaseLabel">
         <property name="text">
          <string>Database</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="databaseTypeLabel">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="databaseType">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>SQLite</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="databasePathLabel">
           <property name="text">
            <string>Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="databasePath"/>
         </item>
         <item>
          <widget class="QPushButton" name="databaseBrowse"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="logLabel">
         <property name="text">
          <string>Log</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="logTypeLabel">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="logType">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Plain</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HTML</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="logPathLabel">
           <property name="text">
            <string>Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="logPath"/>
         </item>
         <item>
          <widget class="QPushButton" name="logBrowse"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="logSizeLabel">
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="logSize">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="logRotateLabel">
           <property name="text">
            <string>Rotate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="logRotate">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="logDebugLabel">
           <property name="text">
            <string>Debug</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="logDebug"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="logInfoLabel">
           <property name="text">
            <string>Info</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="logInfo"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="logWarningLabel">
           <property name="text">
            <string>Warning</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="logWarning"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="logErrorLabel">
           <property name="text">
            <string>Error</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="logError"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="logCriticalLabel">
           <property name="text">
            <string>Critical</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="logCritical"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="shoutboxLabel">
         <property name="text">
          <string>Shoutbox</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="shoutbotLogLabel">
           <property name="text">
            <string>Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="shoutboxLog"/>
         </item>
         <item>
          <widget class="QLineEdit" name="shoutboxPath"/>
         </item>
         <item>
          <widget class="QPushButton" name="shoutboxBrowse"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="sigLabel">
           <property name="text">
            <string>Server Signature</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sig"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Security</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="sslLabel">
         <property name="text">
          <string>SSL</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="sslEnabledLabel">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sslEnabled"/>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="sslCertLabel">
           <property name="text">
            <string>Certificate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="sslCertPath"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="sslCertBrowse"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="sslKeyLabel">
           <property name="text">
            <string>Key</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="sslKeyPath"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="sslKeyBrowse"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="authDatabaseLabel">
         <property name="text">
          <string>Auth Database</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="authDatabaseTypeLabel">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="authDatabaseType">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>SQLite</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="authPathLabel">
           <property name="text">
            <string>Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="authPath"/>
         </item>
         <item>
          <widget class="QPushButton" name="authBrowse"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="permsLabel">
         <property name="text">
          <string>Default Permissions</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="adminPermLabel">
           <property name="text">
            <string>Admin</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="adminPerm"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="view_dbPermLabel">
           <property name="text">
            <string>View DB</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="view_dbPerm"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="modify_dbPermLabel">
           <property name="text">
            <string>Modify DB</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="modify_dbPerm"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="admin_dbPermLabel">
           <property name="text">
            <string>Admin DB</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="admin_dbPerm"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="adminLabel">
         <property name="text">
          <string>Admin User</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="adminUserPath">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="adminUser"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="saltsLabel">
         <property name="text">
          <string>Salts</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="imageSaltLabel">
           <property name="text">
            <string>Image</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="imageSalt"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="passwordSaltLabel">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="passwordSalt"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>CAPTCHA</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="fontPathLabel">
           <property name="text">
            <string>Font Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fontPath"/>
         </item>
         <item>
          <widget class="QPushButton" name="fontBrowse"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lettersLabel">
         <property name="text">
          <string>Letters (comma separated)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="letters"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Updates</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="clientLabel">
         <property name="text">
          <string>Client</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="windowsClientLabel">
           <property name="text">
            <string>Windows</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="windowsClient"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="macClicentLabel">
           <property name="text">
            <string>Mac OS X</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="macClient"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="linuxClientLabel">
           <property name="text">
            <string>Linux</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="linuxClient"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="updaterLabel">
         <property name="text">
          <string>Updater</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="windowsUpdaterLabel">
           <property name="text">
            <string>Windows</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="windowsUpdater"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="macUpdaterLabel">
           <property name="text">
            <string>Mac OS X</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="macUpdater"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="linuxUpdaterLabel">
           <property name="text">
            <string>Linux</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="linuxUpdater"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
